starport_template/lib/utils/node_info.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+import 'package:starport_template/utils/base_env.dart';
+
+class NodeInfo {
```

Comment:
currently the name suggests its a data class holding info about node, while it's actually a class used for fetching/loading data, so lets name it accordingly
```suggestion
class NodeInfoLoader {
```